logging/rust/src/logging.rs

```diff
@@ -110,6 +110,14 @@ impl LoggingSender<wasmbus_rpc::actor::prelude::WasmHost> {
         )?;
         Ok(Self { transport })
     }
+
+    /// Constructs a client for actor-to-actor messaging
+    /// using the recipient actor's public key
+    pub fn to_actor(actor_id: &str) -> Self {
```

Comment:
I'm not sure if it makes sense to expose a `to_actor` function in the logging interface when there's never(?) a viable use case for actor-to-actor communication on the logging interface. That's always an actor making use of the `wasmcloud:builtin:logging` interface.